src/Bridges/lazybridgeoptimizer.jl

```diff
@@ -35,6 +35,9 @@ function _dist(b::LazyBridgeOptimizer, F::Type{<:MOI.AbstractFunction}, S::Type{
     end
 end
 
+# Compute the sum of `Int`.
+int_sum(f::Function, x) = isempty(x) ? 0 : sum(f, x)
```

Comment:
This naming is confusing because nothing in the code implies that the function works only for `Int`s. What about using `reduce(+,  _dist(b, C[1], C[2]) for C in added_constraint_types(BT, F, S), init=0)` instead of defining a helper function?
(Compat provides this version of `reduce` in 0.6.)

Reply:
It didn't work on Julia v0.6 but I didn't with the Compat prefix